core/frontend/services/settings/index.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+    get routes() {
```

Comment:
Could be more specific/explicit if it was called `routesFile`? When it's only called `routes` it looks a bit confusing when used because "route(s)" is already a term used in express

Reply:
Understand the confusion around `routes` name, maybe `dynamicRoutes` would be a bit less ambiguous? 

Just for reference, we have routes file saved in the following path: `content/settings/routes.yaml`

Naming whole thing routes would give it a similar feeling: `settings.routes.get()` `settings.routes.setFromFilePath` :thinking:

Reply:
I think an all round naming of `dynamicRouting`, `dynamic-routing`, etc. would be better - it's explicitly referencing the feature and there's no confusion between the dynamic routing configuration and all of the other `routes.js` files.

Reply:
perfecto :ok_hand: